tests/formats_test/selinux_test.py

```diff
@@ -1,96 +1,54 @@
+# pylint: disable=unused-import
 import os
-import selinux
-import tempfile
+from unittest.mock import patch, ANY
 import unittest
+import selinux
 
 import blivet
-from tests import loopbackedtestcase
 import blivet.formats.fs as fs
-from blivet.size import Size
 
 
 @unittest.skipUnless(selinux.is_selinux_enabled() == 1, "SELinux is disabled")
```

Comment:
Please remove the above line since these tests do not depend on selinux being enabled.